docs/development/intro.rst

```diff
@@ -166,14 +166,15 @@ more details here: :ref:`setup_cython`. If you want remove the generated files
 run ``make clean``.
 
 For the development it is also convenient to fork and set up the
-:ref:`dev_gammapy-extra`:
+:ref:`dev_gammapy-extra`, as well as declaring some environment variables:
 
 .. code-block:: bash
 
     # Fork the gammapy-extra repository on GitHub, https://github.com/gammapy/gammapy-extra
     cd code
     git clone https://github.com/[your-github-username]/gammapy-extra.git
     export GAMMAPY_EXTRA=$PWD/gammapy-extra
+    export GAMMAPY_DATA=$PWD/gammapy-extra/datasets
```

Comment:
This assumes that all datasets fill be in `gammapy-extra/datasets`. Very soon that will not be the case: #1827 
So better to directly teach devs to have a separate `GAMMAPY_DATA`, just like users, and to have `GAMMAPY_DATA` point there, no?

Reply:
Ok.
Done.